README.md

```diff
@@ -16,51 +16,99 @@ If you're having trouble getting set up, reach out to us at [our mailing list](h
+1. Install [App Engine](https://cloud.google.com/appengine/docs)
+	* Specifically use the [Python SDK](https://cloud.google.com/appengine/downloads#Google_App_Engine_SDK_for_Python)
+	* Run the installer and allow it make symbolic links (it might ask you to enter your root password)
 2. Get the latest version of The Blue Alliance
-	* First make sure you have forked TBA by clicking on "Fork" in the top right of the page
-	* Run `git clone git://github.com/username/the-blue-alliance.git` where username is your GitHub username
+	* Fork TBA by clicking on "Fork" in the top right of [its GitHub page](https://github.com/the-blue-alliance/the-blue-alliance)
+	* Run `git clone git://github.com/USERNAME/the-blue-alliance.git` where _USERNAME_ is your GitHub username, or use GitHub's Windows or OS X app to clone it to your computer
 	* For detailed instructions see [the GitHub guide on contributing](https://guides.github.com/activities/contributing-to-open-source/index.html#contributing)
-3. Import the project into Google App Engine Launcher
-	* By default TBA uses port **8088**, make sure your local setup is consistent with this
-4. Run the app in App Engine
-5. Get some test data: In your terminal console, from the `the-blue-alliance` directory, run the following command
-	* `paver setup` (Make sure that you already have [Paver installed](#paver-commands))
-6. You should now have a basic development installation!
+3. Install [numpy](http://www.numpy.org/)
+	* Download [the latest version of numpy](http://sourceforge.net/projects/numpy/files/latest/download?source=files)
+	* Unpack the archive
+	* cd into the archive
+	* Run `setup.py install` or `python setup.py install`
+4. Install [Node.js](https://nodejs.org/) which includes [Node Package Manager](https://www.npmjs.org/)
+5. Install LESS via Node Package Manager
+	* `npm install -g less`
+6. Install Paver
+	* `easy_install -U Paver`
+	* or `pip install -U Paver`
+7. Run `paver setup`
+
+Run a local dev server
+----------------------
+1. Import the project into Google App Engine Launcher
+	* Open App Engine Launcher
+	* File > Add Existing Application...
+	* Set the Application Path to your `the-blue-alliance` directory
+	* Set port **8088**
+2. Run the app in App Engine Launcher and view its Logs window
+3. You should now have a basic development installation!
 	* Visit [localhost:8088](http://localhost:8088) to see your local version of The Blue Alliance
-	* Visit [localhost:8088/admin/debug](http://localhost:8088/admin/debug) to run more commands and populate your install with extra data
+	* Also see [localhost:8088/admin/](http://localhost:8088/admin/)
+4. Get some data into the local server
+	* NOTE: These steps are intended for local dev servers. They might also work on a deployed server if you set DEBUG to True.
+	* NOTE: The /admin/ page's buttons like "Create Test Teams" and "Create Test Events" are obsolete. They try to access decomissioned usfirst.org data feeds.
```

Comment:
Actually Create Test Events still works. You just need to have teams in the database before you run it.